### PR TITLE
Add SAML/light  request/response conversion for Connector

### PR DESCRIPTION
**Pull request by jiri-janousek**
Ref #11

### CODE REVIEW

File: eidas_node/saml.py [hunk 1]
```diff
@@ -111,11 +117,61 @@ def from_light_request(cls: Type[SAMLRequestType], light_request: LightRequest,
...
+                               citizen_country_code=self.citizen_country_code,
+                               relay_state=self.relay_state)
+        root = self.document.getroot()
+        if root.tag != Q_NAMES['saml2p:AuthnRequest']:
+            raise ValidationError({get_element_path(root): 'Wrong root element: {!r}'.format(root.tag)})
+
+        request.id = root.attrib.get('ID')
+        request.provider_name = root.attrib.get('ProviderName')
+
+        for elm in root:
```
Comment (tpazderka) [inline eidas_node/saml.py]: This is far too complex and needs to be rewritten. It should be possible to use `find` from lxml.

File: eidas_node/tests/test_saml.py [hunk 2]
```diff
@@ -111,11 +113,67 @@ def test_from_light_request_invalid_id(self):
         with self.assert_validation_error('id', "Light request id is not a valid XML id: '0day'"):
             SAMLRequest.from_light_request(request, 'test/destination', datetime(2017, 12, 11, 14, 12, 5, 148000))
 
+    def test_create_light_request_success(self):
+        self.maxDiff = None
+        with cast(TextIO, (DATA_DIR / 'saml_request.xml').open('r')) as f:
+            data = f.read()
+
+        saml_request = SAMLRequest(parse_xml(data), 'CA', 'relay123')
+        self.assertDictEqual(
```
Comment (tpazderka) [inline eidas_node/tests/test_saml.py]: `assertEqual`?

Comment (fenryxo) [reply eidas_node/tests/test_saml.py]: Fixed.

File: eidas_node/saml.py (hunk 1 above)
Comment (fenryxo) [reply eidas_node/saml.py]: Ported to `find` and `findall`.

File: eidas_node/tests/test_saml.py [hunk 3]
```diff
@@ -233,36 +326,44 @@ def test_create_light_response_with_unsupported_sub_status(self):
...
-    def test_create_eidas_attribute_known_attribute(self):
-        root = Element('whatever')
-        elm = create_eidas_attribute(root, 'http://eidas.europa.eu/attributes/naturalperson/CurrentFamilyName', True)
-        self.assertIs(elm, root[0])
-        expected = Element('whatever')
-        SubElement(expected, Q_NAMES['eidas:RequestedAttribute'], {
+class TestCreateAttributeElmAttributes(SimpleTestCase):
+    def test_create_attribute_elm_attributes_known_attribute(self):
+        name = 'http://eidas.europa.eu/attributes/naturalperson/CurrentFamilyName'
+        self.assertEquals(create_attribute_elm_attributes(name, None), {
```
Comment (tpazderka) [inline eidas_node/tests/test_saml.py]: `assertEquals` is deprecated.

Comment (fenryxo) [reply eidas_node/tests/test_saml.py]: I've fixed this in new `TestCreateAttributeElmAttributes` code and the rest will be addresed in #40.

**Pull request merged**